### DISCUSSION TITLE
Custom text in requiresConfirmation modal

### DISCUSSION THREAD

**Discussion by jwhoami** [filamentphp/filament#10102] [Help] [answered]
Filament v3

i have an action with requiresConfirmation

Is it possible to add custom text to the requiresConfirmation modal dialog?

Thanks

**Comment by leandrocfe** [comment, marked as answer]
Yes, take a look
https://filamentphp.com/docs/3.x/actions/modals/#customizing-the-modals-heading-description-and-submit-action-label

**Reply by jwhoami** [reply]
Nice. Thanks

**Comment by yakou32** [comment]
That's nice indeed :)

I was wondering  if it was possible to add line breaks, or even better, some markdown styling...

That would be awesome

**Reply by pragus99** [reply]
Yes, You can use `new HtmlString()`
`->modalDescription(new HtmlString('Actual Amount is big than Remaining Amount.<br> Are you sure you would like to do this?'))`

**Comment by felipe-silveira-dev** [comment]
If you are in v2 use the method ->modalSubheading('your text here')
